src/lib/React3.jsx

```diff
@@ -5,7 +5,7 @@ import * as THREE from 'three';
 import React3Renderer from './React3Renderer';
 import propTypeInstanceOf from './utils/propTypeInstanceOf';
 
-const { PropTypes } = React;
+import PropTypes from 'react/lib/ReactPropTypes';
```

Comment:
It's failing a lint check - absolute imports should come before relative imports. Just move this line up.